projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -62,4 +63,7 @@
     <PackageReference Include="System.IO.Pipelines" Version="7.0.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework) == 'netstandard2.0'">
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
```

Comment:
nit: why not the latest? You'll need it for sure for metrics anyway,!

Reply:
Forgot to update :) will do

Reply:
Updated